examples/filler-values-of-a-list/filler_values_of_a_list.md

```diff
@@ -8,130 +8,31 @@ Lists can be initialized with the help of filler values. These are the default v
-<td>object</td>
-<td><code>new T()</code></td>
-<td>if T is a class, where T is the type descriptor for the object, and the
-static type of T's init method allows no arguments and does not include error</td>
-</tr>
-<tr>
-<td>stream</td>
-<td>empty stream</td>
-<td></td>
-</tr>
-<tr>
-<td><code>xml</code></td>
-<td><code>xml``</code></td>
-<td></td>
-</tr>
-<tr>
-<td>built-in subtype of <code>xml</code></td>
-<td><code>xml``</code></td>
-<td>if this belongs to the subtype, i.e. if the subtype is
-<code>xml:Text</code></td>
-</tr>
-<tr>
-<td>singleton</td>
-<td>the single value used to specify the type</td>
-<td></td>
-</tr>
-<tr>
-<td rowspan="2">union</td>
-<td><code>()</code></td>
-<td>if <code>()</code> is a member of the union</td>
-</tr>
-<tr>
-<td>the filler value for basic type B</td>
-<td>if all members of the union belong to a single basic type B,
-and the filler value for B also belongs to the union</td>
-</tr>
-<tr>
-<td><code>T?</code></td>
-<td><code>()</code></td>
-<td></td>
-</tr>
-<tr>
-<td><code>any</code></td>
-<td><code>()</code></td>
-<td></td>
-</tr>
-<tr>
-<td><code>anydata</code></td>
-<td><code>()</code></td>
-<td></td>
-</tr>
-<tr>
-<td><code>byte</code></td>
-<td><code>0</code></td>
-<td></td>
-</tr>
-<tr>
-<td>built-in subtype of <code>int</code></td>
-<td><code>0</code></td>
-<td></td>
-</tr>
-<tr>
-<td><code>json</code></td>
-<td><code>()</code></td>
-<td></td>
-</tr>
-</table>
+Type descriptor                | Filler value                                      | When available
+---------------                | ------------                                      | ---------------
+`()`                           | `()` 	                                           |
+`boolean`                      | `false` 	                                       |
+`int`                          | `0` 	                                           |
+`float`                        | `+0.0f` 	                                       |
+`decimal`                      | `+0d` 	 	                                       |
+`string`                       | `""` 	                                           |
+array or tuple type descriptor | `[]` 	                                           | if that is a valid constructor for the type
+map or record type descriptor  | `{}` 	                                           | if that is a valid constructor for the type
+`readonly & T`                 | the filler value for `T` constructed as read-only | if that belongs to the type
+`table`                        | empty table (with no rows) 		               |
+`object`                       | `new T()` 	                                       | if `T` is a class, where `T` is the type descriptor for the object, and the static type of `T`'s init method allows no arguments and does not include error
+`stream`                       | empty stream 		                               |
+`xml`                          | ` xml`` `		                                   |
```

Comment:
\` xml`` \`	 may not work as expected. I tried several ways, but nothing seems to work on IntelliJ preview. Need to check in pre-prod.